### ISSUE TITLE
getScanResult returning empty

### ISSUE THREAD

**Issue by zgmunshi** [parsonsmatt/WifiWizard#96]
I am trying to use this plugin, but i am facing some issues and finding it difficult to implement it.

1) **listNetworks** is giving me my visited all the networks not the current networks

2) getScanResult is returning empty.

Can anyone guide me what i may be doing wrong.

Thanks

**Comment by emcniece** [comment]
I am experiencing this as well. My app is Ionic 2 RC0 (typscript), and I am testing this by using `ionic run android` on my device. At this time, I am accessing the `WifiWizard` object directly through the Chrome Developer Tools during a `chrome://inspect` session.

Next step will be to attempt a bundle and install directly on the device to eliminate potential interference.

**Comment by emcniece** [comment]
Read through https://github.com/hoerresb/WifiWizard/issues/90 when you get a chance.

Tl;dr: enable Location permission for the app manually via _Settings_ -> _App_ -> _My App_ -> _Permissions_
